samcli/local/apigw/local_apigw_service.py

```diff
@@ -272,6 +238,9 @@ def _add_catch_all_path(self, methods, path, route):
                 payload_format_version=route.payload_format_version,
                 is_default_route=True,
                 stack_path=route.stack_path,
+                authorizer_name=route.authorizer_name,
+                authorizer_object=route.authorizer_object,
+                use_default_authorizer=route.use_default_authorizer,
```

Comment:
Previously, the catch all (default) path did not copy over the authorizers, this fixes that.